### ISSUE TITLE
CJS build is broken, using a dependency that is ESM only

### ISSUE THREAD

**Issue by manucorporat** [atomiks/rehype-pretty-code#92]
When running `rehype-pretty-code` in CJS is fails because of dependency `unified` is ESM only. So, it can not be `require`d
```
failed to load config from /Users/manualmeida/repos/builderio/qwik/packages/docs/vite.config.ts
error during build:
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/manualmeida/repos/builderio/qwik/node_modules/.pnpm/unified@10.1.2/node_modules/unified/index.js from /Users/manualmeida/repos/builderio/qwik/node_modules/.pnpm/rehype-pretty-code@0.9.9_shiki@0.14.2/node_modules/rehype-pretty-code/dist/rehype-pretty-code.cjs not supported.
```

**Comment by atomiks** [comment]
`unified` is not a direct dependency of the package, if you want to support CJS you need to have `unified` v9 installed

**Comment by atomiks** [comment]
This was actually a mistake with externalizing `unified` in the conversion to TS, should be fixed in `v0.9.11`
